incident.go

```diff
@@ -653,4 +653,57 @@ func (c *Client) ManageIncidentAlertsWithContext(ctx context.Context, incidentID
 	return &result, nil
 }
 
-/* TODO: Create Status Updates */
+// IncidentStatusUpdate is a status update for the specified incident.
+type IncidentStatusUpdate struct {
+	ID        string    `json:"id,omitempty"`
+	Message   string    `json:"message,omitempty"`
+	CreatedAt string    `json:"created_at,omitempty"`
+	Sender    APIObject `json:"sender,omitempty"`
+}
+
+// CreateIncidentStatusUpdateResponse is returned from the API as a response to creating an incident status update.
+type CreateIncidentStatusUpdateResponse struct {
+	IncidentStatusUpdate IncidentStatusUpdate `json:"status_update"`
+}
+
+// CreateIncidentStatusUpdateWithContext creates a new status update for the specified incident.
+func (c *Client) CreateIncidentStatusUpdateWithContext(ctx context.Context, id string, statusUpdate IncidentStatusUpdate) (*IncidentStatusUpdate, error) {
+	d := map[string]string{
+		"message": statusUpdate.Message,
+	}
+
+	h := map[string]string{
+		"From": statusUpdate.Sender.Summary,
+	}
+
+	resp, err := c.post(ctx, "/incidents/"+id+"/status_updates", d, h)
+	if err != nil {
+		return nil, err
+	}
+
+	var result CreateIncidentStatusUpdateResponse
+	if err = c.decodeJSON(resp, &result); err != nil {
+		return nil, err
+	}
+
+	return &result.IncidentStatusUpdate, nil
+}
+
+// CreateIncidentStatusUpdateWithResponse creates a new status update for the specified incident.
+//
+// Deprecated: Use CreateIncidentStatusUpdateWithContext instead.
+func (c *Client) CreateIncidentStatusUpdateWithResponse(id string, statusUpdate IncidentStatusUpdate) (*IncidentStatusUpdate, error) {
```

Comment:
What's the purpose of this method?

Reply:
I was trying to prepare the method sets like other old methods. But I can understand that we don't need this because of your helpful comments. I'll remove this.